pykern/pkcli/fmt.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+u"""run black formatter on file/dir
+
+:copyright: Copyright (c) 2013-2018 Bivio Software, Inc.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkdebug import pkdp
+import pykern.pksubprocess
+
+
+def edit(path):
+    cmd = ['black', f'{path}']
+    pykern.pksubprocess.check_call_with_signals(cmd)
+
+
+def diff(path, path_expect=None):
+    cmd = ['git', 'diff', '--no-index', f'{path}']
```

Comment:
This should be below the call to edit. Keep definition and use together. However, I think you could actually use `black --diff --no-color`, or even `black --check` , which would be sufficient. Maybe this should just be called fmt.diff and fmt.check to parallel the names black uses.

Reply:
How about `_diff()` is called if `--check` detects diff and the public method is `check_diff()` ?

Reply:
The purpose of check is to be used only in automation. If there's a problem, the solution is to just run fmt, not to see the differences. I realized that knowing the differences wasn't interesting, since formatter will fix everything up. The bug is that the developer committed without running the formatter first.

Reply:
Ok, so separate `check()` and `diff()` functions?

Reply:
Yes.